flink-core/src/main/java/org/apache/flink/configuration/ResourceManagerOptions.java

```diff
@@ -56,14 +56,14 @@
 			" Its not possible to use this configuration key to define port ranges.");
 
 	/**
-	 * Percentage of heap space to remove from containers (YARN / Mesos), to compensate
+	 * Percentage of heap space to remove from containers (YARN / Mesos/ Kubernetes), to compensate
```

Comment:
Also need to adjust the descriptions in `ConfigConstants#CONTAINERIZED_HEAP_CUTOFF_RATIO`?

Reply:
Nice catch. I will also update the description of `ConfigConstants#CONTAINERIZED_HEAP_CUTOFF_RATIO`.